database/seed-PSQL.js

```diff
@@ -0,0 +1,84 @@
+require('dotenv').config();
+const { Client } = require('pg');
+const { arrayGenerator } = require('./dataGenerator');
+const path = require('path');
+
+const fs = require('fs');
+const csv = require('fast-csv');
+
+let timerStart;
+let timeElapsed;
+
+const csvWriter = () => {
+  const ws = fs.createWriteStream(path.join(__dirname, '/test.csv'));
+  const options = {
+    headers: true,
+    transform: (row) => {
+      return {
+        id: row.id,
+        photos: JSON.stringify(row.photos)
+      };
+    }
+  };
+  return new Promise((resolve, reject) => {
+    csv.write(arrayGenerator(), options).pipe(ws)
+      .on('finish', () => {
+        resolve();
+      })
+      .on('error', (error) => {
+        console.log('ERROR creating array', error);
+        reject();
+      });
+  });
+}
+
+const deleteFile = () => {
+  return new Promise((resolve, reject) => {
+    fs.unlink(path.join(__dirname, 'test.csv'), (err) => {
+      if (err) {
+        console.log('error deleting file');
+        reject();
+      } else {
+        resolve();
+      }
+    });
+  });
+}
+
+const client = new Client({
+  user: process.env.PGUSER,
+  host: process.env.PGHOST,
+  database: process.env.PGDATABASE,
+  password: process.env.PGPASSWORD,
+  port: process.env.PGPORT
```

Comment:
need some insight into what .env vars are needed in readme. also tell user to create the database too, i just ran it and it seeded, but went into my public scema database.

Reply:
```
# create database
$> createdb firebnb-galleries
```
Specify in your .env file
```
PGUSER=
PGHOST=
PGDATABASE=firebnb-galleries
PGPASSWORD=
PGPORT=
```